### DISCUSSION TITLE
update to 15.3 doesn't update html

### DISCUSSION THREAD

**Discussion by slashbv** [jomjol/AI-on-the-edge-device#2590] [General]
Hi,
I have tried to update from version 13.0.4 to 15.3.0 but the html is not updated. firmware was updated fine. Any idea how to fix html ?

**Comment by caco3** [comment]
just retry it.

Also try to do an in-between update to https://github.com/jomjol/AI-on-the-edge-device/releases/tag/v14.0.3

If it still fails, maybe the SD-Card is corrupted.
